ta/ta.mk

```diff
@@ -16,6 +16,10 @@ ifeq ($(CFG_TA_MBEDTLS_SELF_TEST),y)
 $(sm)-platform-cppflags += -DMBEDTLS_SELF_TEST
 endif
 
+ifeq ($(CFG_SYSTEM_PTA),y)
```

Comment:
These lines shouldn't be needed, CFG-variables should propagate automatically via `ta-mk-file-export-vars-$(sm)`